### DISCUSSION TITLE
Enhance Awesome-CV with extra Awesome colors

### DISCUSSION THREAD

**Discussion by frijhee** [posquit0/Awesome-CV#397] [Ideas]
I have put together a collection of twenty (20) additional colors to expand the list of existing eight (8) defined Awesome colors. I would like to contribute this enhancement to the project. 

Impacted files will include `awesome-cv.cls` and all `*.tex` files.

Here's an example of some code I'd like to add to `awesome-cv.cls`:
```
% PURPLE family of Awesome colors
\definecolor{awesome-royal-purple}{HTML}{68478D}
\definecolor{awesome-advent-purple}{HTML}{4B365F}
\definecolor{awesome-plum-purple}{HTML}{9C50B6}
\definecolor{awesome-amethyst}{HTML}{9063CD}
\definecolor{awesome-lavender}{HTML}{9873AC}
``` 

The original Awesome colors would be maintained in their own family as:
```
% ORIGINAL Awesome colors
\definecolor{awesome-emerald}{HTML}{00A388}
\definecolor{awesome-skyblue}{HTML}{0395DE}
\definecolor{awesome-red}{HTML}{DC3522}
\definecolor{awesome-pink}{HTML}{EF4089}
\definecolor{awesome-orange}{HTML}{FF6138}
\definecolor{awesome-nephritis}{HTML}{27AE60}
\definecolor{awesome-concrete}{HTML}{95A5A6}
\definecolor{awesome-darknight}{HTML}{131A28}
```
Thanks in advance for considering my offer to contribute.

Cheers, John

**Comment by OJFord** [comment]
Thanks @frijhee, and for your email - sorry I didn't get back to you before you figured it out in #398.
